libs/ui/src/reports/write_in_adjudication_report.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  AnyContest,
+  CandidateContest,
+  Election,
+  Tabulation,
+} from '@votingworks/types';
+import styled, { ThemeProvider } from 'styled-components';
+import { unique } from '@votingworks/basics';
+import { ReportSection, TallyReport, TallyReportColumns } from './tally_report';
+import { LogoMark } from '../logo_mark';
+import { Prose } from '../prose';
+import { TallyReportMetadata } from './tally_report_metadata';
+import { ContestWriteInSummaryTable } from './contest_write_in_summary_table';
+import { makeTheme } from '../themes/make_theme';
+
+const ZeroList = styled.div`
+  break-inside: avoid;
+  h3 {
+    margin-bottom: 0.2rem;
+  }
+  ul {
+    margin: 0;
+    margin-left: 1rem;
+    padding-left: 0;
+    list-style: none;
+  }
+  li {
+    margin: 0.3rem 0;
+    font-style: italic;
+  }
+`;
+
+export interface WriteInAdjudicationReportProps {
+  election: Election;
+  electionWriteInSummary: Tabulation.ElectionWriteInSummary;
+  generatedAtTime: Date;
+  isOfficialResults: boolean;
+}
+
+export function WriteInAdjudicationReport({
+  election,
+  electionWriteInSummary,
+  generatedAtTime,
+  isOfficialResults,
+}: WriteInAdjudicationReportProps): JSX.Element {
+  const statusPrefix = isOfficialResults ? 'Official' : 'Unofficial';
+
+  const allWriteInContests = election.contests.filter(
+    (c): c is CandidateContest => c.type === 'candidate' && c.allowWriteIns
+  );
+
+  // treating `undefined` as "none" here
+  const relevantPartyIds = unique(allWriteInContests.map((c) => c.partyId));
+
+  return (
+    // must wrap in theme so it's available in printing environment
+    <ThemeProvider theme={makeTheme({ sizeMode: 's' })}>
+      <TallyReport data-testid="write-in-tally-report">
+        {relevantPartyIds.map((partyId) => {
+          const party = election.parties.find((p) => p.id === partyId);
+          const electionTitle = party
+            ? `${party.fullName} ${election.title}`
+            : election.title;
+          const partyWriteInContests = allWriteInContests.filter(
+            (c) => c.partyId === partyId
+          );
+          const nonZeroWriteInContests: AnyContest[] = [];
+          const zeroWriteInContests: AnyContest[] = [];
+          for (const contest of partyWriteInContests) {
+            const contestWriteInSummary =
+              electionWriteInSummary.contestWriteInSummaries[contest.id];
+            if (contestWriteInSummary && contestWriteInSummary.totalTally > 0) {
+              nonZeroWriteInContests.push(contest);
+            } else {
+              zeroWriteInContests.push(contest);
+            }
+          }
+          const sectionKey = partyId || 'none';
+
+          return (
+            <ReportSection
+              key={sectionKey}
+              data-testid={`write-in-tally-report-${sectionKey}`}
+            >
+              <LogoMark />
+              <Prose maxWidth={false}>
```

Comment:
nit: This is probably just copied from the pre-existing reports I believe in the current state, `<Prose maxWidth={false}>` is equivalent to a regular div, so worth trying to drop if it doesn't affect the formatting of the page

Reply:
it does change the formatting, but actually brings it more in line with the other report (makes header slightly more pronounced), so I think it looks better. dropped it in 52353c2e6